person.rb

```diff
@@ -10,6 +10,7 @@ def initialize(age, name = 'Unknown', parent_permission: true)
     @name = name
     @age = age
     @parent_permission = parent_permission
+    @rentals = []
```

Comment:
- [ ] like i mentioned above kindly add a method for rental here as well so you can achieve the 'has-many' association
eg 
```ruby
def add_rental(book, date)
    Rental.new(date, book, self)
  end
```